### DISCUSSION TITLE
Questions about backtest results?

### DISCUSSION THREAD

**Discussion by graceyangfan** [nautechsystems/nautilus_trader#724] [Q&A] [answered]
Helllo,recently I look a little deep into the backtest results for crypto_ema_cross_ethusdt_trade_ticks.py
I have some questions,
(1)how was realized_pnl  compute,I used```(pos_open_avg_price`- pos_closed_avg_price)*quantity - commsion_rate*(pos_open_avg_price+pos_closed_avg_price)*quantity``` to compute it,but it is different with nautilus_results.
(2)I did not see how Pnl is compute from  realized_pnl,the sum of realized_pnl is much bigger even subtract the commision

**Comment by cjdsellers** [comment]
This probably relates to the fact that commissions have been getting double counted for both single and multi currency accounts.
The commission was being subtracted once in the PnL calculation, and then again in the account balance update.

This has now been rectified with https://github.com/nautechsystems/nautilus_trader/commit/ddee268b7dda9abf33c286b380ff6a1fbb2f2e0f on develop.

**Comment by graceyangfan** [comment, marked as answer]
@cjdsellers I check it with the latest develop version,realized_pnl is the same,but the realized pnl sum is different from PNL in the report,does I loss something for computing total pnl ?

**Reply by cjdsellers** [reply]
I'm not sure which metrics you're referring to. Total PnL in the backtest report includes commissions, PnL on positions doesn't include commissions - so you could try accounting for that.

There's also an argument to completely standardize this too.

**Reply by graceyangfan** [reply]
@cjdsellers realized pnl does include commision,the realized_pnl in position_report is total pnl in current state
